.github/workflows/linter.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ]  You have some failed Linter tests, Please make sure all linter test pass after you set it up. Linters help check against best practice and code style guideline violation and also improves the overall readability of your code